packages/suite/src/middlewares/wallet/storageMiddleware.ts

```diff
@@ -106,7 +106,7 @@ const storageMiddleware =
             }
 
             case SUITE.UPDATE_SELECTED_DEVICE:
-                if (isDeviceRemembered(action.payload)) {
+                if (isDeviceRemembered(action.payload) && action.payload.mode !== 'bootloader') {
```

Comment:
```suggestion
                if (isDeviceRemembered(action.payload) && action.payload.mode === 'normal') {
```
just to be safe, there are more modes that we dont want to store: initialize -> after wipe device + seedless -> should not happen in suite but...

Reply:
changed and squashed because of a change of commit message